src/Components/Aspire.Confluent.Kafka/AspireKafkaConsumerExtensions.cs

```diff
@@ -55,35 +55,35 @@ public static void AddKafkaConsumer<TKey, TValue>(this IHostApplicationBuilder b
     public static void AddKeyedKafkaConsumer<TKey, TValue>(this IHostApplicationBuilder builder, string name)
```

Comment:
Can you also add a blurb to https://github.com/dotnet/aspire/blob/main/src/Components/README.md#configuration about this new pattern?

Reply:
I added:

> - Each component should support binding arguments from the configuration section and from named configuration as defined below. Settings bound from named configuration should override those bound from integration-specific configuration.